tools/emboss/main.nf

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env nextflow
+
+// Specify DSL2
+nextflow.enable.dsl=2
+
+// Process definition
+process emboss_seqret {
+    tag "${meta.sample_id}"
+
+    publishDir "${params.outdir}/${opts.publish_dir}",
+        mode: "copy",
+        overwrite: true,
+        saveAs: { filename ->
+                      if (opts.publish_results == "none") null
+                      else filename }
+
+    container "quay.io/biocontainers/emboss:6.6.0--h847b9ba_3"
+
+    input:
+        val opts
+        tuple val(meta), path(in_seq)
+
+    output:
+        tuple val(meta), path ("*"), emit: out_seq
```

Comment:
Do you really want wild cards here or can you be more specific about the files which will come out of this and the module below?

Reply:
`seqret` is for converting between a bunch of different file formats, so the input format and output format both vary. There are tons of supported formats: http://emboss.sourceforge.net/docs/themes/SequenceFormats.html

Not sure how to account for all of these. Currently, I only need it to convert a multiple alignment FASTA into PHYLIP, but that is not very forward-thinking.